internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -191,14 +189,15 @@ type Secret struct {
 }
 
 type Event struct {
-	EventID          sdktypes.UUID  `gorm:"uniqueIndex;type:uuid"`
-	IntegrationID    *sdktypes.UUID `gorm:"index"`
-	IntegrationToken string         `gorm:"index"`
-	EventType        string         `gorm:"index:idx_event_type_seq,priority:1;index:idx_event_type"`
-	Data             datatypes.JSON
-	Memo             datatypes.JSON
-	CreatedAt        time.Time
-	Seq              uint64 `gorm:"primaryKey;autoIncrement:true,index:idx_event_type_seq,priority:2"`
+	EventID       sdktypes.UUID  `gorm:"uniqueIndex;type:uuid"`
+	IntegrationID *sdktypes.UUID `gorm:"index"`
+	ConnectionID  *sdktypes.UUID `gorm:"index"`
```

Comment:
plz enforce foreign key for ConnectionID and add the relevant test

Reply:
i added a comment like Integration has, not sure about the status of ENG-590? Let's discuss.

Reply:
do you want me to fix this? I mean foreign key for ConnectionID and the relevant test?

Reply:
@itay ConnecitonID is mandatory, right?
If so it should be sdktypes.UUID and not *sdktypes.UUID

Reply:
fixed